app/serializers/person_serializer.rb

```diff
@@ -11,43 +15,23 @@ class PersonSerializer < ActiveModel::Serializer
     :gender_additional_information,
   )
 
-  has_one :ethnicity, serializer: EthnicitySerializer
-  has_one :gender, serializer: GenderSerializer
-
-  SUPPORTED_RELATIONSHIPS = %w[ethnicity gender].freeze
-
-  def assessment_answers
+  attribute :assessment_answers do |object|
     object.latest_profile&.assessment_answers || []
   end
 
-  def identifiers
-    [police_national_computer, prison_number, criminal_records_office].compact
-  end
+  attribute :identifiers do |object|
+    %i[police_national_computer prison_number criminal_records_office].each_with_object([]) do |identifier, array|
```

Comment:
How do you feel about using `acc` to indicate the idiomatic name for the accumulator in a fold function body?

Reply:
Rather than `array` you mean?

Reply:
Yeah

Reply:
No big pressure :)

Reply:
I think I prefer the name `array` TBH

Reply:
Okey poke